Tests/StreamChatTests/APIClient/Endpoints/PollsEndpoints_Tests.swift

```diff
@@ -0,0 +1,251 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+@testable import StreamChat
+@testable import StreamChatTestTools
+import XCTest
+
+final class PollsEndpoints_Tests: XCTestCase {
```

Comment:
It seems that we are missing PollPayload_Tests? Where we parse a real json response?

Reply:
added